lib/msal-browser/src/request/EndSessionPopupRequest.ts

```diff
@@ -17,7 +17,7 @@ import { PopupWindowAttributes } from "./PopupWindowAttributes";
  * - popupWindowAttributes  - Optional popup window attributes. popupSize with height and width, and popupPosition with top and left can be set.
  * - logoutHint             - A string that specifies the account that is being logged out in order to skip the server account picker on logout
  */
-export type EndSessionPopupRequest = Partial<CommonEndSessionRequest> & {
+export type EndSessionPopupRequest = Partial<Omit<CommonEndSessionRequest, "tokenQueryParameters">> & {
```

Comment:
@Robbie-Microsoft - the 2 session requests should have this omitted (you have already done it)